cloudify_agent/installer/operations.py

```diff
@@ -40,13 +41,10 @@ def create(cloudify_agent, installer, **_):
             ctx.logger.info('Creating Agent {0}'.format(
                 cloudify_agent['name']))
             try:
-                response = installer.runner.run_script(script_path)
-                output = response.std_out
-                if output:
-                    for line in output.splitlines():
-                        ctx.logger.info(line)
-            except CommandExecutionError as e:
-                ctx.logger.error(str(e))
+                installer.runner.run_script(script_path)
+            except (CommandExecutionError, CommandExecutionException):
```

Comment:
`CommandExecutionError` means that we didn't even get to execute the (potentially remote) command.
`CommandExecutionException` means that the (potentially remote) command ran, but encountered a problem.